js/foam/browser/u2/BrowserView.js

```diff
@@ -19,431 +19,360 @@ CLASS({
   name: 'BrowserView',
   extends: 'foam.u2.View',
   requires: [
-    'foam.browser.BrowserConfigU2',
-    'foam.browser.u2.StackView',
-    'foam.input.touch.GestureManager',
-    'foam.input.touch.TouchManager',
-    'foam.u2.ElementParser',
+    'foam.u2.DAOController',
+    'foam.u2.DAOCreateController',
+    'foam.u2.PropertyView',
+    'foam.u2.SearchBorder',
     'foam.u2.md.ActionButton',
-    'foam.u2.md.Checkbox',
     'foam.u2.md.Input',
     'foam.u2.md.Select',
-    'foam.u2.md.SharedStyles',
-    'foam.u2.md.TextArea'
   ],
-
   imports: [
-    'gestureManager',
-    'touchManager',
+    'document',
+    'headerColor',
+    'menuFactory',
+    'selection$',
+    'stack',
   ],
   exports: [
-    'gestureManager',
-    'stack',
-    'touchManager',
+    'data as dao',
```

Comment:
I somehow missed that.  I don't think anything should be exporting data as dao, its not used anywhere.  The ObjectReplicator isn't importing dao, you're passing it in explicitly in the create arguments.